### PR TITLE
sdb 0.10.1

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

File: Library/Formula/sdb.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "glib"
+
+  def install
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system bin/"sdb", testpath/"d", "hello=world"
+    assert_equal "world\n\n", `#{bin}/sdb #{testpath}/d hello`
```
Comment (dunn) [inline Library/Formula/sdb.rb]: You should be able to use `shell_output` here instead of backticks.

File: Library/Formula/sdb.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Sdb < Formula
+  desc "Ondisk/memory hashtable based on CDB."
+  homepage "http://www.radare.org/"
```
Comment (bfontaine) [inline Library/Formula/sdb.rb]: Isn’t there a homepage dedicated to Sdb?

Comment (dholm) [inline Library/Formula/sdb.rb]: I couldn't find one. :/

File: Library/Formula/sdb.rb (hunk 1 above)
Comment (dholm) [inline Library/Formula/sdb.rb]: Thanks, fixed!

File: Library/Formula/sdb.rb (hunk 2 above)
Comment (dunn) [inline Library/Formula/sdb.rb]: Let's use https://github.com/radare/sdb

**Comment by dholm** [comment]
@dunn Fixed.

**Comment by mikemcquaid** [comment]
Thanks again @dholm!

**Pull request closed without merging**